spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -326,7 +326,10 @@ public SparkContext createSparkContext() {
       }
     }
     pythonLibUris.trimToSize();
-    if (pythonLibs.length == pythonLibUris.size()) {
+
+    // Distribute two libraries(pyspark.zip and py4j-*.zip) to workers
+    // when spark version is less than or equal to 1.4.1
+    if (pythonLibUris.size() == 2) {
       conf.set("spark.yarn.dist.files", Joiner.on(",").join(pythonLibUris));
```

Comment:
I think it always discard user provided `spark.yarn.dist.files`. Is it okay to do?